### ISSUE TITLE
Rails 5.2 Compatibility 

### ISSUE THREAD

**Issue by mrsweaters** [prey/gdpr_rails#6]
Hey there, was curious to know if you were able to get this working with Rails 5.2. I'm getting the following error:
```shell
Bundler could not find compatible versions for gem "rails":
  In snapshot (Gemfile.lock):
    rails (= 5.2.0)

  In Gemfile:
    rails (= 5.2.0)

    gdpr_rails was resolved to 0.2.1, which depends on
      rails (~> 5.1.4)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

**Comment by michelson** [comment]
Hi , try the master branch. The compatibility was merged into master but
not released yet to rubygems. I hope to release it on Monday

Best

**Comment by mrsweaters** [comment]
Thanks!

**Comment by EtienneDepaulis** [comment]
This issue could be closed via #4 :)

**Comment by michelson** [comment]
released under gem version 0.2.2. thanks
